homeassistant/components/fully_kiosk/entity.py

```diff
@@ -30,3 +34,28 @@ def __init__(self, coordinator: FullyKioskDataUpdateCoordinator) -> None:
                 (CONNECTION_NETWORK_MAC, coordinator.data["Mac"])
             }
         self._attr_device_info = device_info
+
+    async def mqtt_subscribe(
+        self, event: str | None, event_callback: CALLBACK_TYPE
+    ) -> CALLBACK_TYPE | None:
+        """Subscribe to MQTT for a given event."""
+        data = self.coordinator.data
+        if (
+            event is not None
+            and mqtt.mqtt_config_entry_enabled(self.hass)
+            and data["settings"]["mqttEnabled"]
+        ):
```

Comment:
Invert the logic here for easier to read code:
```suggestion
        if (
            event is None
            or not mqtt.mqtt_config_entry_enabled(self.hass)
            or not data["settings"]["mqttEnabled"]
        ):
            return
```